templates/bridgeserver2.yaml

```diff
@@ -896,6 +899,13 @@ Resources:
                 - - "arn:aws:s3:::"
                   - !Ref ProcessedHealthDataBucket
                   - "/*"
+  ParticipantFileBucket:
```

Comment:
nobody can access this bucket because there is no associated policy for it. You need to create a policy for it so something can access it.  take a look at the [RawHealthDataBucketPolicy](https://github.com/Sage-Bionetworks/BridgeServer2-infra/blob/develop/templates/bridgeserver2.yaml#L797) as an example,

Reply:
BridgeServer uses an IAM user which has AmazonS3FullAccess, so it will be able to access this bucket.